pkg/block/fetcher.go

```diff
@@ -94,18 +107,22 @@ func newSyncMetrics(reg prometheus.Registerer) *syncMetrics {
 		[]string{duplicateMeta},
 		[]string{markedForDeletionMeta},
 	)
+	m.modified = extprom.NewTxGaugeVec(reg, prometheus.GaugeOpts{
+		Subsystem: fetcherSubSys,
+		Name:      "modified",
+		Help:      "Number of block metadata that modified",
```

Comment:
modified... what? (:

Reply:
It's actually specified `Number of block metadata`, I can rephrase it.